config/install/field.field.media.instagram.field_instagram_shortcode.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.media.field_instagram_shortcode
+    - media_entity.bundle.instagram
+id: media.instagram.field_instagram_shortcode
+field_name: field_instagram_shortcode
+entity_type: media
+bundle: instagram
+label: 'Instagram Shortcode'
+description: ''
```

Comment:
I'll add the descriptions but it won't be visible. Shortcode field is hidden in form display and the url field comes with its own description.